test-app/types/ember-data/types/registries/model.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Catch-all for ember-data.
+ */
```

Comment:
it seems we don't need `ember-data` here :)

Reply:
yeah, came from blueprint. removed `ember-data` from `devDependencies` but forgot this place. removed!